duckduckgo/iOS
# Add Network Protection Settings item

Task/Issue URL: https://app.asana.com/0/0/1205045707803897/f
Tech Design URL:
CC:

**Description**:

As part of the work to test a barebones implementation of the `PacketTunnelProvider` providing Network Protection on iOS, we need to add an item in the main iOS Settings through which to access it. As this should currently only be available to internal users and also not even have code included in production builds, this is hidden behind both runtime and compile-time feature flags.



**Steps to test this PR**:
1. Check out this branch
2. Build with Debug configuration
3. [Authenticate as internal user](https://app.asana.com/0/0/1205045707803899/f)
4. Navigate to the Settings screen.
5. **You should see the Network Protection Settings item**
6. Tap the item
7. It should navigate to the Network Protection activity view (initially empty)

<!—
Before submitting a PR, please ensure you have tested the combinations you expect the reviewer to test, then delete configurations you *know* do not need explicit testing.

Using a simulator where a physical device is unavailable is acceptable.
—>

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
